tests/FINDOLOGIC/Export/Tests/ItemTest.php

```diff
@@ -34,23 +35,27 @@ private function getMinimalItem()
         return $item;
     }
 
-    /**
-     * @expectedException \FINDOLOGIC\Export\Data\EmptyElementsNotAllowedException
-     */
     public function testAddingEmptyAttributesCauseException()
     {
-        $item = $this->getMinimalItem();
-        $attribute = new Attribute('empty attribute', []);
-        $item->addAttribute($attribute);
+        try {
+            $item = $this->getMinimalItem();
+            $attribute = new Attribute('empty attribute', []);
+            $item->addAttribute($attribute);
```

Comment:
Test does not fail if no exception is thrown. Needs a `$this->fail(...)` here.

Reply:
Done